src/applications/vaos/reducers/appointments.js

```diff
@@ -28,12 +28,24 @@ export default function appointmentsReducer(state = initialState, action) {
         ...state,
         confirmedStatus: FETCH_STATUS.loading,
       };
-    case FETCH_CONFIRMED_APPOINTMENTS_SUCCEEDED:
+    case FETCH_CONFIRMED_APPOINTMENTS_SUCCEEDED: {
+      const confirmed = action.data.appointmentRequests.filter(
+        req => req.status === 'Booked',
+      );
+      confirmed.sort((a, b) => {
+        if (a.appointmentType < b.appointmentType) {
```

Comment:
I don't think it's mentioned in the ticket, but I believe @rtwell  wanted these sorted by date and time (soonest first).

Reply:
Done. Just thinking out load though. I wonder if the service will return all booked appointments (past, present, and future) or booked appointments after the current date and time? This is related to appointment history if this has to be calculated on the client.